src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -79,24 +124,17 @@ module.exports = kind({
 		]}
 	],
 	bindings: [
-		{from: '.collection', to: '.$.dataList.collection'},
```

Comment:
Hi @DavidUM, it seems like this binding might have been mistakenly removed? This is resulting in the "Popup List" `moon.ContextualPopup` being empty.